client.d.ts

```diff
@@ -38,6 +38,11 @@ declare module "discord.js-light" {
 	interface GuildManager {
 		forge(guild_id: Discord.Snowflake): Discord.Guild
 	}
+
+	interface GuildChannel {
+		fetchOverwrites(): Promise<Discord.Collection<string, Discord.PermissionOverwrites>>
```

Comment:
should be Discord.Snowflake instead of string

Reply:
oops forgot about those :L